app/models/setting.rb

```diff
@@ -3,6 +3,7 @@ class Setting < ApplicationRecord
   validates :description, presence: true
   validates :value, presence: true
   validates :code, uniqueness: true
+  validate :valid_format_for_wishlist_domain_extensions, on: %i[create update]
```

Comment:
Are there any other contexts besides `create` and `update`?

Reply:
`create` and `update` are only contexts that require this validation since other methods don't manipulate with data